docs/how-to/how-to-name-secret.md

```diff
@@ -0,0 +1,14 @@
+# How to name a secret
+
+This tutorial describes how to name a secret in the Google Secret Manager.
+
+## Naming convention
+
+1. The secret name should be in the following format: `<tool for which secret was created>-<entity>`. For example: `github-kyma-bot-token`. From the name, it should be clear what the secret contains: a GitHub token for the Kyma bot service account.
```

Comment:
```suggestion
1. The secret name should be in the following format: `<tool for which secret was created>-<component of the tool>-<entity>`. For example: `github-kyma-bot-token`. The component part is optional and can be skipped if the tool does not contain multiple components. From the name, it should be clear what the secret contains: a GitHub token for the Kyma bot service account.
```

I like the format containing part for the tool/service and part for the component in secret name. Why you dropped this approach?

Reply:
In the https://github.com/kyma-project/test-infra/issues/6536 issue is "The name of the secret must not contain any information about where it is used. If such usage information should be added, labels have to be used.", so I think in this case we should use label for that.

Reply:
In my opinion this is not information about where it is used. It's about describing the identity.

If you have to create a service account for github webhook gateway component in security scanners application how you would name it? Is the name `sec-scanners-gh-webhook-sa` wrong? Giving the SA secret name like `sec-scanners-gh-webhook-sa-key` describe it's content in clear way. How we could name it wise, without using the service and component name?

Even using your proposal `tool for which secret was created`, in my example what the tool would be? GCP, sec-scanners or gh-webhook?

By avoiding describing an usage location I understand constructs like `event-ark-sec-scanners-gh-webhook-sa-key` where in the name of secret you try to indicate this is the sec-scanners-gh-webhook-sa key used in event-ark system.